@ecomplus/storefront-components/src/js/BuyTogether.js

```diff
@@ -104,6 +111,16 @@ export default {
       } else {
         this.discount = this.subtotal * this.discountValue / 100
       }
+    },
+
+    getProductQnts (productsIds) {
+      if (productsIds.length) {
+        const productQnts = {}
+        productsIds.slice(0, 3).forEach(id => {
+          productQnts[id] = 1
+        })
+        this.productQnts = productQnts
+      }
     }
```

Comment:
Assim como não fazia sentido `set*` antes porque não setava nada, agora não faz sentido `get*` porque não retorna nada.
Agora o que ele faz é setar o `this.productQnts`